File: 프로그래머스/lv1/42748. K번째수/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        int[] bowl = {};
        int bowlCnt = 0;
        for(int i = 0; i < commands.length; i++){
            bowl = new int[commands[i][1] - commands[i][0] + 1];
            for(int j = commands[i][0] - 1; j < commands[i][1]; j++){
                bowl[bowlCnt] = array[j];
                bowlCnt++;
            }
            Arrays.sort(bowl);
            answer[i] = bowl[commands[i][2] - 1];
            bowlCnt = 0;
        }
        return answer;
    }
}

File: 프로그래머스/lv1/42748. K번째수/K번째수.js
function solution(array, commands) {
    let answer = [];
    let bowl = [];
    let cnt = 0;
    while(cnt < commands.length){
        for(let i = commands[cnt][0] - 1; i < commands[cnt][1]; i++){
           bowl.push(array[i]);
        }
        bowl.sort((a, b) => a-b);
        console.log(bowl);
        answer[cnt] = bowl[commands[cnt][2] - 1];
        bowl = [];
        cnt++;
    }
    
    return answer;
}